frontend/src/API.js

```diff
@@ -0,0 +1,19 @@
+export const APIGET = async () => {
+  const response = await fetch('https://my-json-server.typicode.com/Lyczeq/images/images');
+
+  return await response.json();
+};
+
+export const APIPOST = async (base) => {
```

Comment:
I'm not sure if these two functions' purpose is clear. Please either treat it as general API helpers (wrappers over `fetch`, provide the URL as parameter) or name it to be less generic (e.g. "SendImageToAPI").